src/lib.rs

```diff
@@ -288,8 +288,26 @@ pub mod tests {
         }
     }
 
-    pub fn clean_db(mut client: &mut Client,
-                    test_idx: &str) {
+    pub fn setup_test_data(client: &mut Client, index_name: &str) {
```

Comment:
I moved this here rather than just making it `pub`lic as I think it could be useful for new tests in other locations.